### ISSUE TITLE
年视图高度的问题

### ISSUE THREAD

**Issue by yilylong** [huanghaibin-dev/CalendarView#299]
请问年视图的内容是自适应高度的吗？demo测试貌似并没有，如果不是全屏， 年视图就不能完整显示。请问怎么调整让年视图内容适应高度。目前小屏手机存在显示不完整，而大屏手机可以完整显示的情况。

**Comment by yilylong** [comment]
我想问题是不是这里：
/**
     * 计算相对高度
     *
     * @param context context
     * @param view    view
     * @return 月视图选择器最适合的高度
     */
    private static int getHeight(Context context, View view) {
        WindowManager manager = (WindowManager) context.getSystemService(Context.WINDOW_SERVICE);
        assert manager != null;
        Display display = manager.getDefaultDisplay();
        int h = display.getHeight();
        int[] location = new int[2];
        view.getLocationInWindow(location);
        view.getLocationOnScreen(location);
        return h - location[1];
    }

默认用屏幕尺寸来算 实际我的日历显示并有全屏 导致算出来的年视图高度过高

**Comment by liulilei** [comment]
我也碰到同样的问题，项目是根据CalendarView的其实高度y坐标 使用屏幕高度减掉y展示的，因为我需要放到首页底部有一个tablayout，所以在展示年试图的时候会被遮挡住，解决办法就只能是自己导入源码，然后修改高度了，希望作者能够提供修改高度的配置，这样灵活性更高

**Comment by yilylong** [comment]
@liulilei  看来我们情况一样 。作者如果能把这个方法放出来给调用者来设置就好了，这样就能满足不同需求了，实在没办法就只能改源码了

**Comment by liulilei** [comment]
@yilylong 是的 引入源码改了下  显示是ok的，根据自己需求改下吧，没有一模一样合适的轮子，稍微改下 就满足需求  最后给作者提供的这个库点赞
